src/modules/GroupChannel/components/MessageList/index.tsx

```diff
@@ -135,10 +158,28 @@ export const MessageList = ({
     replyType,
     scrollPubSub,
   } = useGroupChannelContext();
+  const shouldDisplayScrollToBottom = hasNext() || !isScrollBottomReached;
+  const [delayedDisplayScrollToBottom, setDelayedDisplayScrollToBottom] = useState(shouldDisplayScrollToBottom);
   const store = useSendbirdStateContext();
 
   const [unreadSinceDate, setUnreadSinceDate] = useState<Date>();
 
+  // To account for tween states while scrolling, delay the render of the scroll to bottom button
+  useEffect(() => {
+    let timeout;
```

Comment:
when spamming multi-line messages, I noticed that the scroll to bottom button would flicker on more often because of tween states so I just decided to deal with it